python/py-sentence-transformers/Portfile

```diff
@@ -40,9 +40,9 @@ homepage            https://github.com/UKPLab/sentence-transformers
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}/
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  1f348b52103c1a0628e8e9888c9b8e7f500fc91b \
-                    sha256  a7d6487d1f3d3bd58795064f3b0962fb57ff373b2d92c4c4d51baf509ccfa937 \
-                    size    49102
+checksums           rmd160  ca6098dead338864a765f265341759e58720d87a \
+                    sha256  68250e1e272ad7013c879a633deca710bbaf7b8cec4080095e88904b93eed128 \
+                    size    55609
 
 python.versions     37
```

Comment:
does this version also support 3.8 (from the README it suggests so). Please verify and if this is the case add a py38 subport

Reply:
I haven't migrated yet myself, the buildbots will tell us. Thanks for catching. Done.

Also thanks for your extensive review of `py-opencv` etc. I haven't had two cycles to rub together to address this, but it's still on the to do pile…